b_lambda_layer_common/util/list_batcher.py

```diff
@@ -0,0 +1,36 @@
+import json
+import math
+from itertools import islice
+from typing import Any, Iterable, List
+
+
+class ListBatcher:
+    def __init__(self, items: List[Any], batch_size: int) -> None:
+        """
+        Split the given list of any items into designated size batches.
+
+        :param items: A list of any items that should be split into batches.
+        :param batch_size: Size of batch in bytes. Positive non-zero value.
+
+        return No return.
+        """
+
+        self.items = items
+        self.batch_size = batch_size
+
+    def get_batch(self) -> Iterable:
```

Comment:
Misleading function name. "Get batch" indicates you get a single piece of a list, which is not true. Better function name could be "iterate_batches", do you agree?

Reply:
+ DocString